app/components/attachment_button.js

```diff
@@ -200,7 +206,17 @@ class AttachmentButton extends PureComponent {
     };
 
     render() {
-        const {theme} = this.props;
+        const {theme, wrapper, children} = this.props;
+
+        if (wrapper) {
```

Comment:
It would be nice if we could consolidate the behaviour for the attachment modal in a separate file instead of adding two distinct appearances for this component based on its props, but I can't think of any good way to do that right now